examples/chat-with-rendezvous/chat.go

```diff
@@ -119,13 +119,6 @@ func main() {
 		panic(err)
 	}
 
-	// Bootstrap the DHT. In the default configuration, this spawns a Background
-	// thread that will refresh the peer table every five minutes.
-	logger.Debug("Bootstrapping the DHT")
-	if err = kademliaDHT.Bootstrap(ctx); err != nil {
-		panic(err)
-	}
-
 	// Let's connect to the bootstrap nodes first. They will tell us about the
 	// other nodes in the network.
```

Comment:
This shouldn't be needed, we do it in Kubo too but I think this is wrong.

We could pass [`BootstrapPeers`](https://pkg.go.dev/github.com/libp2p/go-libp2p-kad-dht#BootstrapPeers) option so the DHT handles this itself.